src/components/FileList/FileList.js

```diff
@@ -150,6 +150,8 @@ const parsePaths = data => {
   return data.map(item => {
     const folders = item.path.split('/')
 
+    if(!item.facets) item.facets = 'core'
```

Comment:
This would work for FileList component.
Given that, the FacetsRenderer is a standalone component that we will use anywhere in the webapp.
Probably this implementation needs to be assisted with a check in the FacetsRenderer as well